### PR TITLE
Add app shortcuts to what's new in DevTools (Chrome 84)

### PR DESCRIPTION
**Pull request by beaufortfrancois**
What's changed, or what was fixed?
- Add app shortcuts to what's new in DevTools (Chrome 84)

**Target Live Date:** Don't merge before https://github.com/GoogleChrome/web.dev/pull/2879

- [x] This has been reviewed and approved by @kaycebasques
- [x] I have run `npm test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

File: src/content/en/updates/2020/05/devtools.md [hunk 1]
```diff
@@ -171,6 +171,20 @@ References:
...
+App shortcuts help users quickly start common or recommended tasks within a web
+app. The app shortcuts menu is shown only for Progressive Web Apps that are
+installed on the user's desktop or mobile device.
+
+Check out [Get things done quickly with app
+shortcuts](https://web.dev/app-shortcuts) to learn more.
+
+![App shortcuts in the Manifest pane.](/web/updates/images/2020/05/app-shortcuts.png)
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/9fee874360dbc0f891609a8aa1a9f8947b510253 #}
```
Comment (beaufortfrancois) [inline src/content/en/updates/2020/05/devtools.md:186]: @kaycebasques I've added this line to be consistent with others but I'm not sure about it is used.

Comment (kaycebasques) [reply src/content/en/updates/2020/05/devtools.md:186]: LGTM

File: src/content/en/updates/2020/05/devtools.md [hunk 2]
```diff
@@ -171,6 +171,20 @@ References:
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/5f6928b9570ed35c4a0b4c1de9b0d1f62370c636 #}
 
+
+## App shortcuts support in the Manifest pane
```
Comment (kaycebasques) [inline src/content/en/updates/2020/05/devtools.md:175]:
```suggestion
## App shortcuts support in the Manifest pane {: #app-shortcuts }
```

**Comment by beaufortfrancois** [comment]
@petele can you merge and push live?

**Pull request merged**